src/github.com/getlantern/lantern-ui/app/partials/feed.html

```diff
@@ -1,6 +1,6 @@
 <a class="item" target="_blank" href="{{feed.link}}" ng-click="trackFeed(feedsTitle)">
   <div ng-if="feed.image" class="feed-thumbnail">
-    <img ng-src="{{feed.image}}"/>
+    <img ng-src="{{feed.image}}" on-error="hideImage(feed)"/>
```

Comment:
Oops I didn't notice you are commenting @myleshorton . I should use more descriptive name but `on-error` corresponds to the `onError` directive, which knows about the `$scope`, so it can call `$scope.hideImage`. With `onerror` we can only manipulating element properties like `style`.